homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: |
+      Single platform only, so no entity.py file.
+      CoordinatorEntity is used.
+  config-flow-test-coverage: done
```

Comment:
`test_form_updates_host`, filling in the form via the config flow should not overwrite the old one. We have a reconfigure flow for that now.

Reply:
So should we just cancel the flow showing an error?
Should discovery flow (dhcp) update the host?

Reply:
The discovery flow should update the host, since that is useful for DHCP based networks.
We should abort if the unique id is already set, yes.

Reply:
Addressed in #132886